src/main/java/pl/niewiemmichal/underhiseye/model/LaboratoryExamination.java

```diff
@@ -0,0 +1,65 @@
+package pl.niewiemmichal.underhiseye.model;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@RequiredArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class LaboratoryExamination {
+
+    @Id @GeneratedValue
+    private Integer id;
+
+    @Lob
```

Comment:
Drop the @Lob annotation and use @Size + @Column(length = 8000) instead. Do this for every field that uses lob, because in @ClaudiaPolaczek model we use varchar(8000) for big fields